types/rc-time-picker/index.d.ts

```diff
@@ -0,0 +1,54 @@
+// Type definitions for rc-time-picker 3.4
+// Project: http://github.com/react-component/time-picker
+// Definitions by: Frithjof Winkelmann <https://github.com/Hoff97>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
+
+import * as moment from "moment";
+import * as React from 'react';
+
+export {};
+
+interface TimePickerProps {
+    prefixCls: string;
+    clearText: string;
+    disabled: boolean;
+    allowEmpty: boolean;
+    open: boolean;
+    defaultValue: moment.Moment;
+    defaultOpenValue: moment.Moment;
+    value: moment.Moment;
+    placeholder: string;
+    className: string;
+    id: string;
+    popupClassName: string;
+    showHour: boolean;
+    showMinute: boolean;
+    showSecond: boolean;
+    format: string;
+    disabledHours: () => number[];
+    disabledMinutes: (hour: number) => number[];
+    disabledSeconds: (hour: number, minute: number) => number[];
+    use12Hours: boolean;
+    hideDisabledOptions: boolean;
+    onChange: (value: moment.Moment) => void;
+    addon: (timepicker: TimePicker) => JSX.Element;
+    placement: string;
+    transitionName: string;
+    onOpen: (state: { open: boolean }) => void;
+    onClose: (state: { open: boolean }) => void;
+    hourStep: number;
+    minuteStep: number;
+    secondStep: number;
+    focusOnOpen: boolean;
+    inputReadOnly: boolean;
+    inputIcon: React.ReactNode;
+    clearIcon: React.ReactNode;
+}
+
+export default class TimePicker extends React.Component<Partial<TimePickerProps>> {
```

Comment:
This package was declared using `module.exports = ...` or some variant. It should be declared with an `export = ...` like one of the following:

### Variables

```ts
declare var xyz: SomeType;

export = xyz;
```

### Classes

```ts
class Foo {
  // ...
}

export = Foo
```

### Functions

```ts
function foo(): any;

export = foo;
```

To import the entity using a default import, you will need to enable `esModuleInterop`.

[See more here in the FAQ](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/9f9f6430117d025aef62ed56128deb65508edc86/README.md#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default).